tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -253,15 +236,40 @@ def _directory_factory(
             directory_stack_name = created_directory_stacks.get(region, {}).get("directory")
             logging.info("Using directory stack named %s created by another test", directory_stack_name)
         else:
-            directory_stack = _create_directory_stack(
-                cfn_stacks_factory,
-                request,
-                directory_type,
-                region,
-                vpc_stack,
-            )
-            directory_stack_name = directory_stack.name
-            created_directory_stacks[region]["directory"] = directory_stack_name
+            directory_stack_name = ""
+            cfn_client = boto3.client("cloudformation", region_name=region)
```

Comment:
Let's define the logic to retrieve a stack by tag in a dedicate function in a utility script that can be used by other tests. In the near future we may need to reuse this logic to retrieve other pre-existing auxiliary stacks.